internal/business/server/healthcheck_handler.go

```
@@ -68,11 +103,17 @@ func healthcheckHandlerFunc(operation string, cfg *config.Config, ch *healthchec
 
 		w.Header().Set("Content-Type", "application/json")
 
+		rawResponse := ch.Response()
+		response := rawResponse
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (too many statements above if)

(wsl_v5)